make/test.mk

```diff
@@ -32,33 +32,20 @@ get-test-namespace: ./out/test-namespace
 	@echo -n "test-namespace-$(shell uuidgen | tr '[:upper:]' '[:lower:]')" > ./out/test-namespace
 
 .PHONY: test-e2e
-## Runs the e2e tests without coverage
-test-e2e: build build-image e2e-setup
+## Runs the e2e tests locally
+test-e2e: e2e-setup
 	$(info Running E2E test: $@)
-	$(Q)go test ./test/e2e/... \
-		-parallel=1 \
-		${V_FLAG} \
-		-root=$(PWD) \
-		-kubeconfig=$(HOME)/.kube/config \
-		-globalMan ./deploy/test/global-manifests.yaml \
-		-namespacedMan ./deploy/test/namespace-manifests.yaml \
-		-singleNamespace
+	$(eval DEPLOYED_NAMESPACE := $(TEST_NAMESPACE))
```

Comment:
@baijum what is the need to store the `TEST_NAMESPACE` value as `DEPLOYED_NAMESPACE` and then use it in the next line?

Reply:
@DhritiShikhar That `DEPLOYED_NAMESPACE` environment variable is later used in the test. The test was already written before. If you see my change in `test/e2e/component_test.go` in this PR, you can see the location where it is used.  The environment variable is used to identify the namespaces. Later inside the test, a wait is added for the operator deployment to take place.

Reply:
addressed here 
efa601c